nerfstudio/model_components/ray_samplers.py

```diff
@@ -540,6 +540,7 @@ def __init__(
         single_jitter: bool = False,
         update_sched: Callable = lambda x: 1,
         initial_sampler: Optional[Sampler] = None,
+        pdf_sampler: Optional[PDFSampler] = None,
```

Comment:
Please also add this to "Args" in docstring.

Reply:
Done :).